spec/helpers/playerHelpers.test.js

```diff
@@ -0,0 +1,115 @@
+/* eslint-env jest */
+import helpers from '../../client/helpers/playerHelpers.js';
+
+describe('createSongBar', () => {
+  // TODO: Add in a test for edge case where len input < nBars
+  // TODO: addin in a test for different inputs?
+  let heights = '';
+  let max = 0;
+
+  beforeEach(() => {
+    heights = '';
+    max = Math.floor(Math.random() * 30) + 100;
+    const soundLength = Math.floor(Math.random() * 100) + 700;
+    for (let i = 0; i < soundLength; i += 1) {
+      heights += `${Math.floor(Math.random() * max)},`;
+    }
+  });
+
+  test('should return an array of default length (140) of numbers representing song profile', () => {
+    const songProfile = helpers.createSongBar(heights, max);
+    expect(songProfile.length).toBe(140);
+    expect(Math.max(...songProfile)).toBeLessThanOrEqual(1);
+    expect(Math.min(...songProfile)).toBeGreaterThanOrEqual(0);
+  });
+
+  test('should return an array of varied length (170) of numbers representing song profile', () => {
+    const songProfile = helpers.createSongBar(heights, max, 170);
+    expect(songProfile.length).toBe(170);
+    expect(Math.max(...songProfile)).toBeLessThanOrEqual(1);
+    expect(Math.min(...songProfile)).toBeGreaterThanOrEqual(0);
+  });
+});
+
+describe('convertToMinSec', () => {
+  test('convert time in sec to a string format: min:sec', () => {
+    expect(helpers.convertToMinSec(311)).toBe('5:11');
+  });
+  test('add extra zeros when needed to seconds', () => {
+    expect(helpers.convertToMinSec(241)).toBe('4:01');
+  });
+  test('add zeros when needed to minutes', () => {
+    expect(helpers.convertToMinSec(31)).toBe('0:31');
+  });
+  test('force non-numbers to zero', () => {
+    expect(helpers.convertToMinSec('hi')).toBe('0:00');
+  });
+});
+
```

Comment:
is there a case for if the time is longer than an hour?

Reply:
There is not! I'll add that in.